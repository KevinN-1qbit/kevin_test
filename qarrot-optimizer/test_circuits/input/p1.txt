Rotate 2: ZIX
Rotate -2: ZII
Rotate -2: IIX
Rotate 2: XZI
Rotate -2: IZI
Rotate -2: XII
Rotate 2: ZII
Rotate 2: XII
Rotate 2: ZII
Rotate 2: IXZ
Rotate -2: IIZ
Rotate -2: IXI
Rotate 2: XZI
Rotate -2: IZI
Rotate -2: XII
