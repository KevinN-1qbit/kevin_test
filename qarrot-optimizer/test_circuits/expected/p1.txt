Measure -: YZY
Measure +: ZZZ
Measure +: ZIZ